libs/common/src/lib/services/bootstrap.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Injectable } from '@angular/core'
+import {
+  SettingsListResponseApiModel,
+  SiteApiService,
+  UiApiService,
+  UiSettingApiModel,
+} from '@lib/gn-api'
+import { Observable, of } from 'rxjs'
+import { catchError, map, pluck, shareReplay } from 'rxjs/operators'
+import { LogService } from './log.service'
+
+@Injectable({
+  providedIn: 'root',
+})
+export class BootstrapService {
+  private uiConfigurations: Record<string, Observable<UiSettingApiModel>>
+
+  constructor(
+    private siteApiService: SiteApiService,
+    private logService: LogService,
+    private uiService: UiApiService
+  ) {
+    this.uiConfigurations = {}
+  }
+
+  siteInfoReady(): Observable<SettingsListResponseApiModel> {
+    return this.siteApiService.getSiteOrPortalDescription().pipe(
+      map((r) => r),
```

Comment:
what is that for?

Reply:
I saw it in the initial method so I kept it, thought you added that :) but yes it's useless.